index.html

```diff
@@ -34,7 +34,16 @@
         <span class="nav__promo">30 Days Free Return</span>
     </nav>
     <main>
-   
+        <section>
+            <span class="promo">Canon camera</span>
+            <button class="shop__now">Shop now</button>
```

Comment:
"__" (double low dash) describes that something is inside something, I would name it 'shop-now'